### PR TITLE
Adding back italic text (_Text_)

### PR DESCRIPTION
**Pull request by explodingcamera**
Also changed bold to `**Text**` and italic to `*Text*` because thats how github does it (I just think that'd be more familiar for users).

### CODE REVIEW

**Comment by ImExiled** [comment]
I hate to be that guy, but I seriously hate the idea of this. I hate using \* or *\* for bold or italic. I just hate it. Like if its going to be added at least have it be a toggle.

**Comment by explodingcamera** [comment]
`*` already is bold, this just adds italic. What are the alternatives?

**Comment by goto-bus-stop** [comment]
Slack has `*bold*`, and `_italic_`, which is a bit more fun, I think.

**Comment by foodbandlt** [comment]
@goto-bus-stop We had `_italic_` in the past, but it interferes with usernames since _ is a legal character.  Same goes for -.

The easiest way would be to use characters that aren't able to be used in usernames (everything not alpha-numeric, -, _).

Slightly harder way, adjust the parser to not parse mentions for formatting tags.  That might be the way to go, honestly.

**Comment by explodingcamera** [comment]
okay, it's now using _ and not parsing mentions anymore

**Comment by foodbandlt** [comment]
@henr-y Sorry to get back to you so late on this.  I don't know how much I care about misuse of the tags, but `This_ is a @User_with_underscores` makes the text between the first and second underscore italic despite the second being part of the username.  I don't care all that much, though I'd like to preserve the username if possible.

TL;DR : This works if the beginning underscore is in the username, but breaks when the ending underscore is in the username.

**Comment by explodingcamera** [comment]
@foodbandlt The other tags can be misused too atm ^^
edit: I'll try to change the regex a bit

**Comment by henr-y** [comment]
You'd need to remove all usernames from msg and add them again after they've been parsed.

**Comment by foodbandlt** [comment]
Ahah, nice.  I didn't check that.  Should be a way to do it with regex.  If you can't get it I'll put some thought into it, heh.

Thanks!  Sorry for being so stringent...
